### ISSUE TITLE
TypeError: startswith first arg must be str or a tuple of str, not bytes (Python 3)

### ISSUE THREAD

**Issue by FlorentF9** [jsvine/pdfplumber#33]
With a lot of different PDF documents I get the folllowing error in Python 3 :

```python
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-35be4098fc3e> in <module>()
----> 1 pdfp.open(files[0])

/usr/local/lib/python3.5/dist-packages/pdfplumber/pdf.py in open(cls, path, **kwargs)
     38     @classmethod
     39     def open(cls, path, **kwargs):
---> 40         return cls(open(path, "rb"), **kwargs)
     41 
     42     def process_page(self, page):

/usr/local/lib/python3.5/dist-packages/pdfplumber/pdf.py in __init__(self, stream, pages, laparams, precision)
     30                 self.metadata[k] = list(map(decode_text, v))
     31             elif isinstance(v, PSLiteral):
---> 32                 self.metadata[k] = decode_text(v.name)
     33             else:
     34                 self.metadata[k] = decode_text(v)

/usr/local/lib/python3.5/dist-packages/pdfplumber/utils.py in decode_text(s)
     63     Adds py3 compatability to pdfminer's version.
     64     """
---> 65     if s.startswith(b'\xfe\xff'):
     66         return six.text_type(s[2:], 'utf-16be', 'ignore')
     67     else:

TypeError: startswith first arg must be str or a tuple of str, not bytes
```

However, I don't get this error in Python 2 so I think it is a compatibility issue.

**Comment by jsvine** [comment]
Thanks for flagging, @FlorentF9! Can you attach/share one of the PDFs that triggers this error?

**Comment by FlorentF9** [comment]
I uploaded an example file. If it can help you, it happens with all the PDFs that were generated by pandoc + pdflatex.

[1.pdf](https://github.com/jsvine/pdfplumber/files/990295/1.pdf)

**Comment by jsvine** [comment]
Thanks! I believe I've fixed this in https://github.com/jsvine/pdfplumber/pull/34, where I've also added a test with your sample PDF. Now available in `v0.5.5`. Let me know, though, if you continue to get these (or related) errors.

**Comment by FlorentF9** [comment]
I tried the new version and now it works, thanks! I will let you know if I have problems with other documents.
